src/app/inventory/store/d2-item-factory.ts

```diff
@@ -569,6 +570,7 @@ export function makeItem(
     masterworkInfo: null,
     infusionQuality: null,
     tooltipNotifications,
+    comparisonName: normalBucket.inWeapons ? stripAdeptSuffix(defs, name) : name,
```

Comment:
I'm kind of nervous about adding this in to every item build - it's a bunch of string operations. I'd feel better if we could use it only where it's needed.

Could we maybe generate the suffixes statically in d2ai?

Reply:
I don't love this either and probably wouldn't want to land it this way, mostly just seeing if folks have better ideas. Generating it in d2ai would be ideal, but I don't think d2ai supports manifests/defs for multiple languages at the same time (yet).

Reply:
we have not done anything like that in d2ai yet, but it should not be hard

https://github.com/delphiactual/destiny2-open-i18n/blob/main/src/findDifferentStrings.ts#L10-L15